frontend/resources/locales.json

```
@@ -21,7 +21,7 @@
     "used-in" : [ "src/app/main/ui/auth/login.cljs:161", "src/app/main/ui/auth/register.cljs:138" ],
     "translations" : {
       "en" : "Create demo account",
-      "fr" : "Vous voulez juste essayer?",
+      "fr" : "Vous voulez juste essayer ?",
```

I made a mistake when I first translated this (wrong phrase). Should be:
```suggestion
      "fr" : "Créer un compte de démonstration",
```

---

This is addressed in https://github.com/penpot/penpot/pull/591/commits/341d6a1b008a1e813e89ba128e3fb5210b76fde2.